services/QuillLMS/client/app/bundles/Staff/components/evidence/llmPromptDatasets/individualTrial/individualTrial.tsx

```diff
@@ -43,6 +43,7 @@ const IndividualTrial: React.FC<RouteComponentProps<ActivityRouteProps>> = ({ ma
   const [histogram, setHistogram] = React.useState<any>(null)
   const [trial, setTrial] = React.useState<TrialInterface>(null)
   const [gEvals, setGEvals] = React.useState<GEvalInterface[]>(null)
+  const [gEvalScores, setGEvalScores] = React.useState<any>(null)
```

Comment:
I think you want to use the `GEvalScoreInterface` you defined in `evidenceInterfaces` here.